include/enum_quda.h

```diff
@@ -175,6 +177,13 @@ extern "C" {
     QUDA_INVALID_RESIDUAL = QUDA_INVALID_ENUM
   } QudaResidualType;
 
+  // Which basis to use for CA algorithms
+  typedef enum QudaCABasis_s {
+    QUDA_POWER_BASIS,
+    QUDA_CHEBYSHEV_BASIS,
+    QUDA_INVALID_BASIS = QUDA_INVALID_ENUM
+  } QudaCABasis; 
```

Comment:
clang-format suggested the following change:


```
-  } QudaCABasis; 
+  } QudaCABasis;
```

<!-- identifier: clang-format -->
<!-- File: include/enum_quda.h -->